tests/baselines/reference/parseBigInt.errors.txt

```diff
@@ -0,0 +1,124 @@
+tests/cases/compiler/parseBigInt.ts(51,20): error TS2735: Operator '+' cannot be applied to type '123n'.
+tests/cases/compiler/parseBigInt.ts(52,23): error TS2735: Operator '+' cannot be applied to type '291n'.
+tests/cases/compiler/parseBigInt.ts(56,25): error TS1005: ',' expected.
+tests/cases/compiler/parseBigInt.ts(57,25): error TS1005: ',' expected.
+tests/cases/compiler/parseBigInt.ts(58,22): error TS1005: ',' expected.
+tests/cases/compiler/parseBigInt.ts(59,28): error TS1005: ',' expected.
+tests/cases/compiler/parseBigInt.ts(60,23): error TS1177: Binary digit expected.
+tests/cases/compiler/parseBigInt.ts(61,20): error TS1178: Octal digit expected.
+tests/cases/compiler/parseBigInt.ts(62,20): error TS1125: Hexadecimal digit expected.
+tests/cases/compiler/parseBigInt.ts(63,26): error TS2304: Cannot find name '_123n'.
+tests/cases/compiler/parseBigInt.ts(64,30): error TS6188: Numeric separators are not allowed here.
+tests/cases/compiler/parseBigInt.ts(65,33): error TS6189: Multiple consecutive numeric separators are not permitted.
+tests/cases/compiler/parseBigInt.ts(69,15): error TS2345: Argument of type '0n' is not assignable to parameter of type '1n | 3n | 2n'.
+tests/cases/compiler/parseBigInt.ts(70,15): error TS2345: Argument of type '0' is not assignable to parameter of type '1n | 3n | 2n'.
+tests/cases/compiler/parseBigInt.ts(70,34): error TS2345: Argument of type '1' is not assignable to parameter of type '1n | 3n | 2n'.
+tests/cases/compiler/parseBigInt.ts(70,53): error TS2345: Argument of type '2' is not assignable to parameter of type '1n | 3n | 2n'.
+tests/cases/compiler/parseBigInt.ts(70,72): error TS2345: Argument of type '3' is not assignable to parameter of type '1n | 3n | 2n'.
+
+
+==== tests/cases/compiler/parseBigInt.ts (17 errors) ====
+    // All bases should allow "n" suffix
+    const bin = 0b101, binBig = 0b101n; // 5, 5n
+    const oct = 0o567, octBig = 0o567n; // 375, 375n

```

Comment:
octals are error right?

Reply:
Only legacy octals, e.g. `0567n`. `0o567n` is allows.

Reply:
But then I don't see test case for that. Please add one.

Reply:
Already have one here: https://github.com/Microsoft/TypeScript/pull/25886/files/0d54f041d66536b5712e9bb5e94f28aa7d430892#diff-f6f1b31177fd1675c39b4b3f7ad83581R80

Reply:
The pointer is wrong. can you please provide correct link Thanks. Unresolving this comment so that i can track this.

Reply:
Ah, sorry about that. Must be because I force-pushed. I'll copy the section here, from the `parseBigInt` test:
````typescript
{ const legacyOct = 0123n; }
````